api-inference-community/docker_images/common/tests/test_api.py

```diff
@@ -17,6 +17,7 @@
 
 ALL_TASKS = {
     "audio-to-audio",
+    "audio-classification",
```

Comment:
Nit: if we are trying to preserve alphabetical order here
```suggestion
    "audio-classification",
    "audio-to-audio",
```